### ISSUE TITLE
Disable basic auth for location /.well-known/acme-challenge/

### ISSUE THREAD

**Issue by ypereirareis** [JrCs/docker-letsencrypt-nginx-proxy-companion#112]
Hi !
Actually, I have preprod environments protected by basic auth.
I would like to generate and renew certs without temporarily remove basic auth.

Maybe something like this:

```
location /.well-known/acme-challenge/ {
    auth_basic off
    allow all;
    root /usr/share/nginx/html;
    try_files $uri =404;
    break;
}
```

**Comment by tmtben** [comment]
Hi @JrCs,
Thanks for your great work.
What about this feature?

**Comment by buchdag** [comment]
Fixed by #232, closing the issue.
